crates/sui-core/src/node_sync/node_follower.rs

```diff
@@ -57,7 +59,7 @@ impl EffectsStakeMap {
     }
 
     // Get the set of authorities who voted for a digest.
-    pub fn voters(&self, digest: &TransactionEffectsDigest) -> BTreeSet<AuthorityName> {
+    pub fn voters(&self, digest: &TransactionEffectsDigest) -> HashSet<AuthorityName> {
```

Comment:
Again are we certain that the use of Hash set does not introduce any non-determinism?